### PR TITLE
Add index correction document

### PR DESCRIPTION
**Pull request by ykadowak**
### Description:

SSIA

### Versions:

- Go Version: 1.21.3
- Docker Version: 20.10.8
- Kubernetes Version: v1.28.2
- NGT Version: 2.1.3

### Checklist:

- [ ] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/main/CONTRIBUTING.md) document and completed [our CLA agreement](https://cla-assistant.io/vdaas/vald).
- [ ] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?

### CODE REVIEW

File: docs/user-guides/index-correction.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+```
+
+## Important Notes
+
+- Processing time  
+  Under conditions of 10 million vectors and agent replica \*10, it takes about 10~20 minutes. The process is O(MN) where M is the number of vector items and N is the number of agent replicas.
+- concurrencyPolicy  
+  `Forbid` is set internally, so a new job will not be created while an existing job is running. In other words, if the process does not finish within the interval specified by the schedule, the next job will not be scheduled.
+- Index operations during correction  
+  Vector operations performed after the start of the index correction job are not considered in that job.
```
Comment (hlts2) [inline docs/user-guides/index-correction.md:37]: Is it better to add a new line for readability? 🤔 

```suggestion
- Processing time  
  Under conditions of 10 million vectors and agent replica \*10, it takes about 10~20 minutes. The process is O(MN) where M is the number of vector items and N is the number of agent replicas.

- concurrencyPolicy  
  `Forbid` is set internally, so a new job will not be created while an existing job is running. In other words, if the process does not finish within the interval specified by the schedule, the next job will not be scheduled.

- Index operations during correction  
  Vector operations performed after the start of the index correction job are not considered in that job.
```

**Review by hlts2 — APPROVED** [review]
LGTM!

File: docs/user-guides/index-correction.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    corrector:
+      enabled: true
+      schedule: "3 6 * * *"
+      suspend: false
+```
+
+## Important Notes
+
+- Processing time  
+  Under conditions of 10 million vectors and agent replica \*10, it takes about 10~20 minutes. The process is O(MN) where M is the number of vector items and N is the number of agent replicas.
```
Comment (kpango) [inline docs/user-guides/index-correction.md:33]: I think O(MN) is worst case.
It would be better to describe about Index Replica.

**Review by kpango — APPROVED** [review]
LGTM

**Review by hlts2 — APPROVED** [review]

**Pull request merged**